reframe/core/logging.py

```diff
@@ -144,7 +144,8 @@ def emit(self, record):
         except OSError as e:
             raise LoggingError('logging failed') from e
 
-        self.baseFilename = os.path.join(dirname, record.check.name + '.log')
+        self.baseFilename = os.path.join(dirname,
```

Comment:
```suggestion
        self.baseFilename = os.path.join(
            dirname, f'{record.__rfm_check__.name}.log'
        )
```

Reply:
The way I have it is equivalent and still fits in a single line.